### PR TITLE
Display: added all the features from 3 files

### PR DESCRIPTION
**Pull request by tannuiscoding**
I have implemented the three `write.py`, `show.py` and `clip.py` in `display.py`.

### CODE REVIEW

File: package/clip.py [hunk 1]
```diff
@@ -5,14 +5,11 @@
 from datetime import datetime
 import glob
 def display(snippet_name, password):
-    # Retrieve the current time in HHMM format
     current_time = datetime.now().strftime("%H%M")
     
-    # Check if the provided password matches the current time
-    if str(password) != current_time:
+    if int(password) != current_time:
```
Comment (jindalpriyanshu101) [inline package/clip.py:10]: This condition will never be true. as you're comparing int `password` with a string `current_time`
Consider typecasting `current_time` to int, OR don't typecast `password` 

```suggestion
    if int(password) != int(current_time):
```

OR Consider this:
```suggestion
    if password != current_time:
```

File: package/show.py [hunk 2]
```diff
@@ -3,10 +3,11 @@
 import subprocess
 import sys
 from datetime import datetime 
+import glob
 
 def display(snippet_name, password, clipboard=None):
     current_time = datetime.now().strftime("%H%M")
-    if str(password) != current_time:
+    if int(password) != current_time:
```
Comment (jindalpriyanshu101) [inline package/show.py:10]: same here

File: package/write.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 def plot(snippet_name,password):
     current_time = datetime.now().strftime("%H%M")
-    if str(password) != current_time:
+    if int(password) != current_time:
```
Comment (jindalpriyanshu101) [inline package/write.py:8]: same here as well

**Review by jindalpriyanshu101 — CHANGES_REQUESTED** [review]
You're comparing different datatypes(string & int) as password check, fix that. i've attached 2 suggestions that you can implement to fix that.

**Comment by jindalpriyanshu101** [comment]
The time is ticking 

image

File: package/show.py (hunk 2 above)
Comment (this-is-yaash) [inline package/show.py:10]:
```suggestion
    if int(password) != int(current_time):
```

File: package/write.py (hunk 3 above)
Comment (this-is-yaash) [inline package/write.py:8]:
```suggestion
    if int(password) != int(current_time):
```

**Review by this-is-yaash — APPROVED** [review]
`display.py` feature is working perfectly. I have updated the arguments to accept `int` instead of `str`.

So far, LGTM! ✅

**Comment by this-is-yaash** [comment]
fixes #77

**Pull request merged**